ngraph_bridge/ngraph_encapsulate_clusters.cc

```diff
@@ -96,6 +97,10 @@ Status EncapsulateClusters(
   // A map from cluster indices to corresponding NGraphEncapsulate nodes.
   std::map<int, Node*> cluster_node_map;
 
```

Comment:
Note to self: Add a comment here on why this is called // reference the ngraph ops for static library

Reply:
yes please add a comment why this file needs to call register_bridge()

Reply:
There is a comment in `ngraph_bridge/ngraph_bridge_registry.h`:
```
// Created a dummy function for bridge specific ops
// so when built static, the linker will include
// all the object files which it would not do normally.
```